azure/vnet/local.tf

```diff
@@ -0,0 +1,60 @@
+locals {
+  default_tags = {
+    ProvisionedBy = "Terraform"
```

Comment:
@filipefigueredo should we start using the new tag or should we plan a one big update in all modules?

Reply:
For now, I think we can leave it there, so we can comply to the same guidelines used in the existing modules. But I guess you are right, we should consider this major refactor after we test / define all desired tags